components/automate-chef-io/content/docs/data-collection.md

```diff
@@ -21,7 +21,7 @@ There are two steps to getting data collection running in Chef Automate:
 
 1. You must first have an API token. You have two options:
 
-  * [Create a new API Token]({{< relref "api-tokens.md#creating-api-tokens" >}}).
+  * [Create a new API Token]({{< relref "api-tokens.md#creating-api-tokens" >}}) and add it to the Ingest policy.
```

Comment:
@susanev How does this addition of "add it to the Ingest policy" play into the second step below of "Once you have an API token, you can either:"? Also, is there documentation for the Ingest policy that can be linked here?

Reply:
@mjingle the 2nd bullet point will automatically add it to the ingest policy for you, if you create an api token you'll need to choose the ingest policy in the dropdown in the create modal.

there's not really separate documentation on each chef-managed policy, other than the table of roles that are associated with the policy. we could link to that i suppose... https://automate.chef.io/docs/roles/#chef-managed-roles

Reply:
@susanev So then, this addition is clarifying a requirement to apply during the API Token creation?

Reply:
If so, then I would recommend this rewording:
```suggestion
  * [Create a new API token]({{< relref "api-tokens.md#creating-api-tokens" >}}) and assign the API token to the Ingest policy during creation.
```

Reply:
@mjingle I think _add_ is the right work here not _assign_ (at least thats the word used in the UI), and they dont necessarily have to do it during creation, but they can do it then.

Reply:
@susanev How about now?
```suggestion
  * [Create a new API token]({{< relref "api-tokens.md#creating-api-tokens" >}}) and add the API token to the Ingest policy, preferably at time of creation.
```